core/new-gui/src/app/workspace/types/execute-workflow.interface.ts

```diff
@@ -37,7 +37,8 @@ export interface LogicalPlan
     operators: LogicalOperator[];
     links: LogicalLink[];
     breakpoints: BreakpointInfo[];
-    cachedOperatorIds: string[];
+    opsToViewResult?: string[];
+    opsToReuseResult?: string[];
```

Comment:
to simplify client code logic, is it better to make them an empty list by default? so that the client does not need to do null checks.